waku/waku_core/message/default_values.nim

```diff
@@ -0,0 +1,7 @@
+
+const
+  ## https://rfc.vac.dev/spec/64/#message-size
+  MaxWakuMessageSize* = 150 * 1024 # Remember that 1 MiB is the PubSub default
+
+  DefaultMaxWakuMessageSizeStr* = "150KiB"
```

Comment:
I wouldn't have two sources of truth (`MaxWakuMessageSize` and `DefaultMaxWakuMessageSizeStr`). How about we use the `str` one as source of truth, and define the other in terms of it by calling `parseMsgSize(DefaultMaxWakuMessageSizeStr)`?